packages/manager/modules/dedicated-universe-components/src/index.js

```diff
@@ -0,0 +1,96 @@
+import angular from 'angular';
+
+import ducBandwidth from './bandwidth';
+import ducBytes from './bytes';
+import ducContract from './contract';
+import ducNotification from './notification';
+import ducPrice from './price';
+import ducTabs from './tabs';
+import ducTranslate from './translate';
+
+import ducCheckboxSwitch from './directives/checkboxSwitch';
+import ducSshKeySwitch from './directives/sshkeySwitch';
+
+import config from './config';
+import user from './user';
+import noDefaultMeansOfPayment from './no-default-means-of-payment';
+import sshkeyMinFilter from './filters/sshkeyMin';
+import validatorService from './validator.service';
+
+const moduleName = 'ngOvhDedicatedUniverseComponents';
+
+angular
+  .module(moduleName, [
+    ducBandwidth,
+    ducBytes,
+    ducContract,
+    ducNotification,
+    ducPrice,
+    ducTabs,
+    ducTranslate,
+    ducCheckboxSwitch,
+    ducSshKeySwitch,
+    user,
+    noDefaultMeansOfPayment,
+  ])
+  .constant('constants', {
```

Comment:
I think those should be kept in dedicated or renamed. Let's say we have an app that requires both dedicated and cloud universe components and both constants have been added as `constants` there will be a conflict 
```suggestion
  .constant('ducConstants', {
```

Reply:
And this is true for all AngularJS things exported in a module. So here, we need to prefix all `component/service/constant/controller/...` declared with `duc`